web/src/components/dashboard/SidebarCardRegisteredForTaxesNudge.tsx

```diff
@@ -32,6 +32,7 @@ export const SidebarCardRegisteredForTaxesNudge = (props: Props): ReactElement =
         card={props.card}
         bodyText={props.card.contentMd}
         headerText={props.card.header}
+        preBodySpanButtonText={props.card.preBodySpanButtonText}
```

Comment:
remove, preBodySpanButtonText should be optional, only add where necessary

Reply:
remove from other cards

Reply:
done